gremlin-go/driver/gorillaTransporter.go

```diff
@@ -100,6 +101,9 @@ func (transporter *gorillaTransporter) Write(data []byte) error {
 			return err
 		}
 	}
+	if len(data) > transporter.connSettings.writeBufferSize {
+		return errors.New("request too large, consider increasing 'WriteBufferSize' configuration or split into smaller requests")
```

Comment:
all errors should be defined in `error_codes.go`
and used like `return newError(err0101ConnectionCloseError)`